modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/bookmark/BookmarkManagerImpl.kt

```diff
@@ -26,7 +26,7 @@ class BookmarkManagerImpl @Inject constructor(
      */
     override suspend fun add(episode: BaseEpisode, timeSecs: Int, title: String): Bookmark {
         // Prevent adding more than one bookmark at the same place
-        val existingBookmark = bookmarkDao.findByEpisodeTime(podcastUuid = episode.podcastOrSubstituteUuid, episodeUuid = episode.uuid, timeSecs = timeSecs)
```

Comment:
It will also consider bookmarks in deleted state and wouldn't allow adding a new bookmark at that time. Is it expected?